### DISCUSSION TITLE
Tokens Migration from old wallet to firefly wallet

### DISCUSSION THREAD

**Discussion by bond2504** [iotaledger/firefly#7298] [Help & Support]
I am migrating tokens to firefly, my 81 seeds are correct and even checksum is correct too,  but the balance on firefly  is 0. why any help.thanks

**Comment by begonaalvarezd** [comment]
Hi, you might need to re-run the balance search multiple times until the balances are found

**Reply by bond2504** [reply]
okay let me try again

**Comment by KaherdinTristan** [comment]
@begonaalvarezd 
- Where in the Firefly app is the button/option to re-run the search? 
If applicable, could you provide a screen shot?

**Reply by begonaalvarezd** [reply]
on october 4th the migration was stopped because we upgraded mainnet, it will be resumed soon, we will provide a new tool for it (I presume you are already aware of it because of your message here https://github.com/iotaledger/firefly/discussions/7672#discussioncomment-8116886) Will close this conversation as it is not relevant anymore
